src/debug-integration.adoc

```diff
@@ -1,17 +1,20 @@
 [#section_debug_integration]
-== Integrating Zcheri_purecap with Sdext
+=== Integrating Zcheri_purecap with Sdext
 
 This section describes changes to integrate the Sdext ISA and
 {cheri_base_ext_name}. It must be implemented to make external debug compatible
 with {cheri_base_ext_name}. Modifications to Sdext are kept to a minimum.
 
-=== Debug Mode
+WARNING: This section is preliminary as no-one has yet built debug support
+ for CHERI-RISC-V so change is likely.
+
+==== Debug Mode
 
 When executing code due to an abstract command, the hart stays in debug mode
 and the rules outlined in Section 4.1 of cite:[riscv-debug-spec]
 apply.
 
-=== Core Debug Registers
+==== Core Debug Registers
```

Comment:
I think the headers within this section are a little off.

Reply:
yeah - there aren't enough levels to make it work - I think it's good enough